lib/devise-guests/version.rb

```diff
@@ -1,3 +1,3 @@
 module DeviseGuests
-  VERSION = "0.8.1"
+  VERSION = "0.8.2"
```

Comment:
The version gets updated during a release, not on this branch.
```suggestion
  VERSION = "0.8.1"
```